### PR TITLE
Add more properties to ThreeDSecureResult

### PR DESCRIPTION
**Pull request by ashkarpetin**

### CODE REVIEW

File: openapi/components/schemas/Transactions/ThreeDSecureResult.yaml [hunk 1]
```diff
@@ -5,20 +5,26 @@ required:
   - authenticated
   - liability
 properties:
+  version:
```
Comment (ashkarpetin) [inline openapi/components/schemas/Transactions/ThreeDSecureResult.yaml:8]: Alternative option: add `is3ds2`

Comment (svensono) [reply openapi/components/schemas/Transactions/ThreeDSecureResult.yaml:8]: I'm a fan of the version more

**Review by subenson — DISMISSED** [review]

File: openapi/components/schemas/Transactions/ThreeDSecureResult.yaml (hunk 1 above)
Comment (adamaltman) [reply openapi/components/schemas/Transactions/ThreeDSecureResult.yaml:8]: The version is much better because there is 2.0 2.1, 2.2 already.

Comment (ashkarpetin) [reply openapi/components/schemas/Transactions/ThreeDSecureResult.yaml:8]: Changed to

```
type: string
    enum:
      - 1.0
      - 2.0
      - 2.1
      - 2.2
```

**Review by adamaltman — DISMISSED** [review]

**Review by iamdevice — DISMISSED** [review]

**Review by wmdmgv — APPROVED** [review]

**Review by slavcodev — APPROVED** [review]

**Review by adamaltman — APPROVED** [review]

**Review by iamdevice — APPROVED** [review]

**Pull request merged**